.gitignore

```diff
@@ -50,3 +50,6 @@ Pods/
 screenshots
 
 .DS_Store
+
+## Plist
+wsdot/*.plist
```

Comment:
Unfortunately, this also captures the project's default `Info.plist` which is needed. Let's change this to specifically ignore the `GoogleServices-Info.plist` file instead.